spec/system/registered_user/changing_password_spec.rb

```diff
@@ -1,55 +0,0 @@
-require 'rails_helper'
```

Comment:
We have removed the link on the account page at this point, but the route and functionality remains. I think if that is the case we shouldn't be deleting the spec, which I presume still works?